### PR TITLE
CustomTimer: Use float time instead of decimal

### PR DESCRIPTION
**Pull request by SmileyAG**

### CODE REVIEW

**Comment by SmileyAG** [comment]
> Why?

Useful when I want to measure with checkpoints for how much some route differences in HLKZ with a bind like:

`bind v "stop;record route;say /practicetp;customtimer 20" // teleport to latest practice checkpoint`

File: cl_dll/hud_customtimer.cpp [hunk 1]
```diff
@@ -34,7 +34,7 @@ int CHudCustomTimer::Draw(float time)
 	UnpackRGB(r, g, b, gHUD.m_iDefaultHUDColor);
 
 	FillRGBA((ScreenWidth / 2) - 50, (gHUD.m_scrinfo.iCharHeight * 4) - 6, (diff / m_flSeconds) * 100, 4, r, g, b, 210);
-	snprintf(str, sizeof(str), "Timer: %d", (int)diff);
+	snprintf(str, sizeof(str), "Timer: %.1f", diff);
```
Comment (YaLTeR) [inline cl_dll/hud_customtimer.cpp:37]:
```suggestion
	snprintf(str, sizeof(str), "Timer: %.01f", diff);
```
I think this is needed for it to show .0

Comment (SmileyAG) [reply cl_dll/hud_customtimer.cpp:37]: It does show even without that change.

**Comment by YaLTeR** [comment]
Thanks

**Pull request merged**